main.nf

```diff
@@ -348,6 +356,8 @@ summary['Reads']        = params.reads
 summary['Fasta Ref']    = params.fasta
 if(params.bwa_index) summary['BWA Index'] = params.bwa_index
 summary['Data Type']    = params.singleEnd ? 'Single-End' : 'Paired-End'
+summary['Skip Collapsing'] = params.skip_collapse ? 'Yes' : 'No'
```

Comment:
Is there a reason why these aren't logical/boolean? It would be more consistent with other `param.skip_*`s

Reply:
These are still booleans, the yes/no is just for logging purposes.

Reply:
:+1: my misunderstanding.